manimlib/scene/scene_embed.py

```diff
@@ -143,8 +143,11 @@ def reload_scene(self, embed_line: int | None = None) -> None:
 
     def auto_reload(self):
         """Enables IPython autoreload for automatic reloading of modules."""
```

Comment:
might want to adjust the docstring to make clear we don't use the `%autoreload` IPython line magic and instead reload user modules with our custom logic